hawkrest/__init__.py

```diff
@@ -92,11 +111,14 @@ def authenticate(self, request):
         # Pass our receiver object to the middleware so the request header
         # doesn't need to be parsed again.
         request.META['hawk.receiver'] = receiver
-        return on_success
+        return self._user_lookup(request, receiver.resource.credentials)
 
     def authenticate_header(self, request):
         return 'Hawk'
 
+    def get_user(self, user_id):
```

Comment:
It's a requirement from Django proper, not DRF. See [the Django docs](https://docs.djangoproject.com/en/1.10/topics/auth/customizing/#writing-an-authentication-backend). At least, I *thought* it was required. Now I'm not so sure, since DRF [doesn't seem to include it in their auth classes](https://github.com/encode/django-rest-framework/blob/master/rest_framework/authentication.py) -- and I can't tell if that's an oversight, or if `get_user()` is deprecated in Django and I missed it.